Packs/ApiModules/Scripts/ZoomApiModule/ZoomApiModule.yml

```diff
@@ -0,0 +1,17 @@
+commonfields:
+  id: ZoomApiModule
+  version: -1
+name: ZoomApiModule
+script: "-"
+subtype: python3
+type: python
+comment: Zoom Client class, provides generic Infrastructure to all Zoom integrations
```

Comment:
Common Zoom code that provides generic Infrastructure, and will be appended to each Zoom integration when it is deployed.